corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -148,6 +148,9 @@ var ReportConfigsViewModel = function (options) {
         $.ajax({
             type: "DELETE",
             url: options.saveUrl + '/' + config._id(),
+            beforeSend: function(xhr, settings) {
```

Comment:
There's `ajaxSetup` that I could have used, but thought this approach was better. I will change this to use `ajaxSetup`.